zio-sql/jvm/src/test/scala/zio/sql/UnaryOpSpec.scala

```diff
@@ -0,0 +1,53 @@
+package zio.sql
+
+import zio.test.{assert, DefaultRunnableSpec, suite, test}
+import zio.test.Assertion.anything
+
+object ProductSchema {
+  val userSql = new Sql { self =>
+    import self.ColumnSet._
+
+    val productTable = (
+      string("id") ++
+        localDate("last_updated") ++
+        string("name") ++
+        int("amount") ++
+        boolean("deleted")
+      ).table("product")
+
+    val id :*: lastUpdated :*: name :*: amount :*: deleted :*: _ = productTable.columns
+
+    val selectAll = select { id ++ lastUpdated ++ amount ++ deleted } from productTable
+
+    val selectDeteltedNotTrue = selectAll.where( deleted.isNotTrue )
```

Comment:
typo ... Detelted

Reply:
Renamed. In fact, I renamed all queries to `query`

(In the different contexts - in the business app - that, most likely, would be an antipattern. But here the name do not give us anything. Actually it forces you to focus on irrelevant business logic. Important is how this logic is implemented. That is only relevant to open source and internal libraries.)